src/components/MessagesDataTable.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <el-table
+    :data="tableData"
+    :default-sort="{ prop: 'date', order: 'descending' }"
+  >
+    <el-table-column
+      v-for="col in tableLabel"
+      :key="col.id"
+      :prop="col.prop"
+      :label="col.label"
+      :sortable="col.isSortable"
+      :column-key="col.prop"
+      :width="col.width"
+    />
+    <el-table-column prop="action" label="">
+      <div class="action-icons">
+        <i class="fa-solid fa-clipboard-check icon"></i>
+        <i class="fa-solid fa-bell icon"></i>
+      </div>
+    </el-table-column>
+  </el-table>
+  <el-pagination layout="prev, pager, next" :total="50"> </el-pagination>
+</template>
+
+<script>
+import { ElTable, ElTableColumn, ElPagination } from 'element-plus';
+import 'element-plus/dist/index.css';
```

Comment:
This is still importing the entire lib css, you need to import only the specfiic files for this component
so the path would be something like `element-plus/components/table` and so on